docs/PayPalNativeCheckout/README.md

```diff
@@ -0,0 +1,170 @@
+# Accepting PayPal Native Checkout Payments
+
+The PayPal Native Checkout module in the PayPal SDK enables PayPal payments via a native UI experience in your app.
+
+Follow these steps to add PayPal Native Checkout payments:
+
+1. [Setup a PayPal Developer Account](#setup-a-paypal-developer-account)
+1. [Add PayPal Native Checkout Module](#add-paypal-native-checkout-module)
+1. [Test and go live](#test-and-go-live)
+
+## Setup a PayPal Developer Account
+
+You will need to set up authorization to use the PayPal Payments SDK. 
+Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID. 
+
+You will need a server integration to create an order to capture funds using the [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). 
+For initial setup, the `curl` commands below can be used as a reference for making server-side RESTful API calls.
+
```

Comment:
We should also mention in this step that they need to fetch an ACCESS_TOKEN from their server. All good if we want this in a separate PR, since it affects all docs.

Reply:
@scannillo this is mentioned in the [general readme](https://github.com/paypal/iOS-SDK/blob/main/README.md) before we redirect them to each module.

Reply:
maybe we should add it to each module? but we would be repeating ourselves